app/views/layouts/application.html.erb

```diff
@@ -7,13 +7,9 @@
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
-
   <body>
-
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
     <%= render './shared/navbar' %>
-    
+    <%= render 'shared/top_banner_section' %>
```

Comment:
The top banner section will be probably different in each of the pages. Also take into account that the homepage doesn't have a banner like the other 3 pages.

If that's the case, I don't think you want to render the banner in this file, but actually in each of the different pages where it belongs :wink: